src/components/GitmojiList/Toolbar/styles.module.css

```
@@ -6,14 +6,36 @@
   margin-top: 1.5rem;
 }
 
-.searchInput {
+.inputWrapper {
+  flex: 1;
+  display: flex;
+  align-items: center;
   border-radius: 4px;
   border: 0;
   box-shadow: 0 2px 4px 0 var(--cardShadow);
-  flex: 1;
+  margin-right: 1rem;
+  position: relative;
+}
+
+.searchInput {
+  background-color: transparent;
+  border: none;
+  flex-grow: 1;
   font-size: 1rem;
   padding: 1rem;
-  margin-right: 1rem;
+}
+
+.kbd {
+  position: absolute;
+  right: 0;
+  align-items: center;
+  border-radius: 3px;
+  border: solid 1px #999;
+  color: #b8b8b8;
+  display: flex;
+  font-family: system-ui;
+  margin: 0 0.5rem;
```

```suggestion
  margin-right: 0.5rem;
```
It's a good practice to never set properties you are not using with CSS